tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java

```diff
@@ -354,7 +354,7 @@ private TiColumnInfo copyColumn(TiColumnInfo col) {
   }
 
   public TiTableInfo copyTableWithRowId() {
-    if (!isPkHandle()) {
+    if (!isPkHandle() && !isCommonHandle) {
```

Comment:
better to add some comments saying the change is made due to behavior shift between non-cluster index and cluster index.

Reply:
This is a bug. The change makes the correct behavior of show _tidb_rowid. Do I need to add comments in this case?

Reply:
I have added comments to clear it

Reply:
> This is a bug. The change makes the correct behavior of show _tidb_rowid. Do I need to add comments in this case?

It is probably because back to non-cluster index days, we do not need to consider that. ;)